src/features/duplicates/hooks/useDuplicatesMutations.tsx

```diff
@@ -0,0 +1,29 @@
+import { dismissedDuplicate, ZetkinDuplicate } from '../store';
+import { useApiClient, useAppDispatch } from 'core/hooks';
+
+type DuplicatesMutationsReturn = {
+  dismissDuplicate: (duplicateId: number) => void;
+};
+
+export default function useDuplicatesMutations(
+  orgId: number
+): DuplicatesMutationsReturn {
+  const apiClient = useApiClient();
+  const dispatch = useAppDispatch();
+
+  const dismissDuplicate = async (duplicateId: number) => {
+    const dismissedDate = new Date().toISOString();
+    await apiClient
+      .patch<ZetkinDuplicate>(
+        `/orgs/${orgId}/people/duplicates/${duplicateId}`,
```

Comment:
I tried this amazing new UI and found that dismissing didn't work, because `/api/` is missing in the URL here. Just making a comment to not forget about it. 😊

Reply:
Fixed!